src/frontend/apps/standalone_site/src/styles/theme.extend.ts

```diff
@@ -32,3 +33,60 @@ export const themeExtend = {
 export const themeBase = theme;
 
 export const getFullThemeExtend = () => deepMerge(themeBase, themeExtend);
+
+export const disableFormTheme = {
+  global: { control: { disabled: { opacity: 1 } } },
+  textInput: {
+    extend: css`
+      cursor: text;
```

Comment:
By default textInput is with `cursor: text` already.

Reply:
Not the input of a disabled select (or not in chrome at least)